jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -444,6 +444,25 @@ private void maybeSendKeyframeRequest(AbstractEndpoint dominantSpeaker)
             return;
         }
 
+        boolean anyEndpointInStageView = false;
+        for (Endpoint otherEndpoint : getLocalEndpoints())
```

Comment:
It seems like we could keep this stat without having to iterate over all the endpoints every time, if we kept a count of how many endpoints in a conference were in stage view?